tests/mentorship_relation/test_api_create_request.py

```diff
@@ -0,0 +1,57 @@
+import unittest
```

Comment:
I loveeee how you did tests for this issue 🎉  And actually are using some parts of the email api that I didn't know, and will have to research a bit! Good job here!